compiler/rustc_lint/src/context.rs

```diff
@@ -856,6 +856,13 @@ pub trait LintContext: Sized {
                         Applicability::MachineApplicable,
                     );
                 },
+                BuiltinLintDiagnostics::NamedArgumentUsedPositionally(positional_arg, named_arg, name) => {
+                    db.span_label(named_arg, "this named argument is only referred to by position in formatting string");
+                    let msg = format!("this formatting argument uses named argument {} by position", name);
+                    db.span_label(positional_arg, msg);
+                    let suggestion = format!("{{{}}}", name);
+                    db.span_suggestion(positional_arg, "use the named argument by name to avoid ambiguity", suggestion, Applicability::MaybeIncorrect);
```

Comment:
```suggestion
                    db.span_suggestion_verbose(
                        positional_arg,
                        "use the named argument by name to avoid ambiguity",
                         format!("{{{}}}", name),
                         Applicability::MaybeIncorrect,
                     );
```

Reply:
Done!